### PR TITLE
Create a template for opening a new issue

### PR DESCRIPTION
**Pull request by digitalentity**

### CODE REVIEW

**Review by DzikuVx — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+### If your issue looks like a hardware fault or a configuration problem please don't raise an issue here.
+
+### Please consider using other user support options such as asking the manufacturer of the hardware you are using, [RCGroups](https://www.rcgroups.com/forums/showthread.php?t=2495732), [Slack](https://inavflight.signup.team) or other user support forums & groups (e.g. facebook).
+
+### Board and Version
+_Board name goes here. If you have a link where you've purchased the board - please include it_
+_Version of INAV used goes here_
+
+_Use CLI command `version` and put it's output here_
```
Comment (stronnag) [inline .github/ISSUE_TEMPLATE.md]: its (not it's)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+### Behavior
+* _Describe the problem_
+* _Steps needed to reproduce the problem_
+* _Expected Results_
+* _Actual Results_
+
+_Is this a new feature request?_
+
+#### Blackbox logs/Config (.ZIP) where problem occurs
+_Go to CLI, execute `dump` command copy it's output to [PasteBin](https://pastebin.com) and provide a link to a paste here.
```
Comment (stronnag) [inline .github/ISSUE_TEMPLATE.md]: its (not it's)

**Review by stronnag — COMMENTED** [review]
tiny grammatical updates

**Comment by digitalentity** [comment]
@stronnag fixed

**Pull request merged**